### PR TITLE
libsvm: add version 333

### PR DESCRIPTION
**Pull request by toge**
### Summary
Changes to recipe:  **libsvm/333**

#### Motivation
There are slight modification in 333.
Since the final release of libsvm is in 2023, it would be worth adding a version to the recipe. 

#### Details
https://github.com/cjlin1/libsvm/compare/v332...v333

---
- [x] Read the [contributing guidelines](https://github.com/conan-io/conan-center-index/blob/master/CONTRIBUTING.md)
- [x] Checked that this PR is not a duplicate: [list of PRs by recipe](https://github.com/conan-io/conan-center-index/discussions/24240)
- [x] Tested locally with at least one configuration using a recent version of Conan

### CODE REVIEW

**Review by perseoGI — DISMISSED** [review]
LGTM

File: recipes/libsvm/all/conanfile.py [hunk 1]
```diff
@@ -12,19 +12,19 @@
 class libsvmConan(ConanFile):
     name = "libsvm"
     description = "Libsvm is a simple, easy-to-use, and efficient software for SVM classification and regression"
+    license = "BSD-3-Clause"
     url = "https://github.com/conan-io/conan-center-index"
     homepage = "https://www.csie.ntu.edu.tw/~cjlin/libsvm/"
-    license = "BSD-3-Clause"
-    topics = "svm", "vector"
+    topics = ("svm", "vector")
     settings = "os", "arch", "compiler", "build_type"
```
Comment (AbrilRBS) [inline recipes/libsvm/all/conanfile.py:19]:
```suggestion
    package_type = "library"
    settings = "os", "arch", "compiler", "build_type"
```

File: recipes/libsvm/all/conanfile.py [hunk 2]
```diff
@@ -12,19 +12,20 @@
 class libsvmConan(ConanFile):
     name = "libsvm"
     description = "Libsvm is a simple, easy-to-use, and efficient software for SVM classification and regression"
+    license = "BSD-3-Clause"
     url = "https://github.com/conan-io/conan-center-index"
     homepage = "https://www.csie.ntu.edu.tw/~cjlin/libsvm/"
-    license = "BSD-3-Clause"
-    topics = "svm", "vector"
+    topics = ("svm", "vector")
+    package_type = "library"
```
Comment (AbrilRBS) [inline recipes/libsvm/all/conanfile.py:19]:
```suggestion

    package_type = "library"
```

**Review by AbrilRBS — APPROVED** [review]
Added a missing `package_type`, thanks @toge!

**Review by perseoGI — APPROVED** [review]
LGTM

**Pull request merged**